src/plugins/controls/common/control_group/types.ts

```diff
@@ -9,10 +9,12 @@
 
 import { DataViewField } from '@kbn/data-views-plugin/common';
 import { ControlLabelPosition, DefaultControlState, ParentIgnoreSettings } from '../types';
+import { CONTROL_CHAINING_OPTIONS } from '../constants';
 
 export const CONTROL_GROUP_TYPE = 'control_group';
 
-export type ControlGroupChainingSystem = 'HIERARCHICAL' | 'NONE';
+export type ControlGroupChainingSystem =
```

Comment:
You should be able to use just `export type ControlGroupChainingSystem =
  keyof typeof CONTROL_CHAINING_OPTIONS;`

Reply:
Discussed offline with @nreese. This is using (typeof CONTROL_CHAINING_OPTIONS)[keyof typeof CONTROL_CHAINING_OPTIONS] to get the values from the object rather than the keys.